libs/src/services/creatorNode/CreatorNodeSelection.ts

```diff
@@ -159,31 +258,31 @@ class CreatorNodeSelection extends ServiceSelection {
   /**
    * Get backups in the form of an array
    */
-  getBackupsList () {
+  getBackupsList() {
     return this.backupsList
   }
 
   /**
    * Get backup timings in the form of an array
    */
-  getBackupTimings () {
+  getBackupTimings() {
     return this.backupTimings
   }
 
   /**
    * Select a primary Content Node
    * @param {string[]} services all healthy Content Node endpoints
    */
-  getPrimary (services) {
+  getPrimary(services: string[]) {
     // Index 0 of services will be the most optimal Content Node candidate
-    return services[0]
+    return services[0] as string
```

Comment:
If `services` is a `string[]` here, why the `as string`?

Reply:
so there is a chance that it's empty array. do you think we should allow it to be string | undefined and have to catch this case in consumers? i went with the cast here to keep from changing the runtime code up too much

Reply:
ahhh gotcha that makes sense. I do think making consumers aware if that's a thing that can happen would be wise, tbh not sure if it happens in practice?